app/src/main/java/com/rentminder/service/HouseholdService.kt

```diff
@@ -9,7 +9,7 @@ import kotlinx.coroutines.withContext
 import retrofit2.awaitResponse
 
 class HouseholdService {
-    suspend fun fetchHouseholds() : List<Household>?{
+    suspend fun fetchHouseholdsList() : List<Household>?{
```

Comment:
changed 'fetchHouseholds()' to 'fetchHouseholdsList()'. This both makes the function name more descriptive, and easier to understand what it is when using it later on/what it is supposed to be returning.